Tries to do network io and fails in config/makefiles/rust.mk
--- firefox-69.0/config/makefiles/rust.mk
+++ firefox-69.0/config/makefiles/rust.mk
@@ -241,16 +241,6 @@ force-cargo-library-build:
 	$(call CARGO_BUILD) --lib $(cargo_target_flag) $(rust_features_flag) -- $(cargo_rustc_flags)
 
 $(RUST_LIBRARY_FILE): force-cargo-library-build
-# When we are building in --enable-release mode; we add an additional check to confirm
-# that we are not importing any networking-related functions in rust code. This reduces
-# the chance of proxy bypasses originating from rust code.
-ifndef DEVELOPER_OPTIONS
-ifndef MOZ_DEBUG_RUST
-ifeq ($(OS_ARCH), Linux)
-	$(call py_action,check_binary,--target --networking $@)
-endif
-endif
-endif
 
 force-cargo-library-check:
 	$(call CARGO_CHECK) --lib $(cargo_target_flag) $(rust_features_flag)
